main/streams/memory.c

```diff
@@ -349,7 +349,7 @@ static ssize_t php_stream_temp_write(php_stream *stream, const char *buf, size_t
 	}
 	if (php_stream_is(ts->innerstream, PHP_STREAM_IS_MEMORY)) {
 		zend_off_t pos = php_stream_tell(ts->innerstream);
-		
```

Comment:
Random whitespace change. I don't care about this, but maybe you do? idk, just pointing it out :)